polaris-react/src/components/IndexTable/tests/IndexTable.test.tsx

```diff
@@ -729,7 +729,7 @@ describe('<IndexTable>', () => {
           </IndexTable>,
         );
 
-        expect(index.findAll(Tooltip)[2].prop('content')).toBe(
```

Comment:
The previous sticky header code was rendering a duplicate tooltip when the first column was sticky and sortable, so to select the "second" column header, the test had to pick out the 3rd Tooltip element.

The changes in this PR remove that unnecessary duplicate Tooltip, and so the test selector can be corrected.